conf/app.routes

```diff
@@ -48,18 +48,21 @@ GET        /:journeyType/confirmation
 
 GET        /:journeyType/exit/:exitType                                                 uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.journey.JourneyExitController.show(journeyType: JourneyType, exitType: JourneyExitType)
 
+
+# Fast Track URLs ----------------------------------------------------------------------------------------------------------
++ nocsrf
+POST        /agents/fast-track                                                          uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.AgentFastTrackController.agentFastTrack
+
 # Client Journey URLs ----------------------------------------------------------------------------------------------------------
 
 GET        /appoint-someone-to-deal-with-HMRC-for-you/:uid/:normalizedAgentName/:taxService             uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.client.StartController.show(uid: String, normalizedAgentName: String, taxService: String)
 
-GET        /authorisation-response/:uid/:taxService/consent-information                                 uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.client.ConsentInformationController.show(uid: String, taxService: String)
+GET        /authorisation-response/:uid/:taxService/consent-information                 uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.client.ConsentInformationController.show(uid: String, taxService: String)
 
-# Fast Track URLs ----------------------------------------------------------------------------------------------------------
-+ nocsrf
-POST        /agents/fast-track                                                          uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.AgentFastTrackController.agentFastTrack
+GET        /authorisation-response/exit/:exitType                                       uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.client.ClientExitController.showClient(exitType: ClientExitType)
```

Comment:
we should add an extra component to the path to avoid the clash with :journeyType/exit/:exitType routes above - if we update the routes to be like `/authorisation-response/exit/auth/:exitType` and `authorisation-response/exit/public/:exitType` then they will not clash